server/index.js

```diff
@@ -31,8 +31,8 @@ app.post("/todo", (request, response) => {
     description: request.body.description,
     completed: request.body.completed,
   };
-  todos.push(newTodo);
-  response.status(201).json(newTodo);
+  const newTodos = [...todos, newTodo];
+  response.status(201).json(newTodos);
```

Comment:
Array.push is a fine "mutability" for now, since it is a list of TODOs stored in memory.
I think there are other pieces of the code that needs to be changed to immutable first.

Meaning: The objects INSIDE this list have to be immutable. Adding/Removing elements (objects) is fine.